From: Jeff Johnson <quic_jjohnson@quicinc.com>
Subject: resource: add missing MODULE_DESCRIPTION()
Date: Wed, 29 May 2024 16:31:58 -0700

Fix the 'make W=1' warning:
WARNING: modpost: missing MODULE_DESCRIPTION() in kernel/resource_kunit.o

Link: https://lkml.kernel.org/r/20240529-md-kernel-resource_kunit-v1-1-bb719784b714@quicinc.com
Signed-off-by: Jeff Johnson <quic_jjohnson@quicinc.com>
Cc: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Cc: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Cc: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 kernel/resource_kunit.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/resource_kunit.c~resource-add-missing-module_description
+++ a/kernel/resource_kunit.c
@@ -149,4 +149,5 @@ static struct kunit_suite resource_test_
 };
 kunit_test_suite(resource_test_suite);
 
+MODULE_DESCRIPTION("I/O Port & Memory Resource manager unit tests");
 MODULE_LICENSE("GPL");
_
